browser-extension/mv2/src/client/js/sessionRecorder.js

```diff
@@ -260,3 +267,32 @@ RQ.SessionRecorder.hideWidget = () => {
 RQ.SessionRecorder.getWidget = () => {
```

Comment:
`getWidget` seems generic. Let's rename all the variables to `manual-mode-widget` and `auto-mode-widget`